api/src/test/scala/hmda/api/processing/HmdaFileUploadSpec.scala

```diff
@@ -2,7 +2,7 @@ package hmda.api.processing
 
 import java.time.Instant
 import akka.testkit.TestProbe
-import hmda.api.processing.HmdaFileUpload.{AddLine, GetState, HmdaFileUploadState}
+import hmda.api.processing.HmdaFileUpload.{ AddLine, GetState, HmdaFileUploadState }
```

Comment:
I made a separate PR (#352) for this, because I didn't notice it here. I'd checked a few other PRs, but didn't happen to look at this one. Is there a better way to do that kind of search?

I am in any case fine with merging this one instead of mine. The whitespace change is a separate commit, so the history will be clear.